lib/spdy-transport/protocol/http2/parser.js

```diff
@@ -153,6 +166,9 @@ Parser.prototype.onFrameBody = function onFrameBody(header, buffer, callback) {
     this.onPriorityFrame(header, buffer, callback);
   else if (header.type === frameType.X_FORWARDED_FOR)
     this.onXForwardedFrame(header, buffer, callback);
+  else if (this._lastHeaderBlock)
+    return callback(this.error(constants.error.PROTOCOL_ERROR,
```

Comment:
If `if`'s body does not fit into one line, it should be enclosed in `{` `}` braces.

If you enclose one body of the `if`-chain, all bodies should be enclosed.